src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -35,15 +35,7 @@
 
 import javax.annotation.Nonnull;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.Locale;
-import java.util.Map;
-import java.util.Set;
-import java.util.Spliterator;
+import java.util.*;
```

Comment:
Better not doing this in production code, especially for such package.

Reply:
Definitively my IDE is not fixed yet :-/